src/core/Status.tsx

```diff
@@ -17,53 +22,46 @@ const indicatorClass = (indicator?: string) => {
   }
 };
 
+export const StatusIcon = ({
+  statusUrl,
+  refreshInterval = 1000 * 60,
+}: {
+  statusUrl: string;
+  refreshInterval?: number;
+}) => {
+  const { data, error, isLoading } = useSWR(statusUrl, fetcher, {
+    refreshInterval,
+  });
+
+  return (
+    <span className="inline-flex items-center h-[1.5rem] p-[0.25rem]">
+      <span
+        className={`w-[1rem] h-[1rem] leading-[1rem] rounded-full ${clsx({ "animate-pulse": isLoading || error })} ${indicatorClass(data?.status?.indicator)}`}
```

Comment:
Good to see `clsx`, but we may as well throw in the rest of it and get rid of the interpolation all together:
```
clsx("w-[1rem] h-[1rem] leading-[1rem] rounded-full", indicatorClass(data?.status?.indicator), { "animate-pulse": isLoading || error })
```
Good thing about this is it helps to get rid of wasted spaces, and gives the IDE linter a fighting chance of preventing massive single lines. Think of `clsx` as "concatenate all this stuff but some of this stuff is conditional"

Reply:
Additionally, do we need a `span` within a `span` here? Seems to me like the same thing could be achieved by taking the inner one, applying `inline-flex` to it, and a margin of 0.25rem (i.e. `m-[0.25rem]` instead of the `p` class), don't need the `items-center` either as there's nothing to center

Reply:
> Additionally, do we need a `span` within a `span` here? Seems to me like the same thing could be achieved by taking the inner one, applying `inline-flex` to it, and a margin of 0.25rem (i.e. `m-[0.25rem]` instead of the `p` class), don't need the `items-center` either as there's nothing to center

There was a reason for this but I cannot remember anymore. The markup is 99% based on the WebComponent from status page, as reverse engineered by Copilot... I'll give it a shot to see what can be done

Reply:
🤷 rem should be consistent, but yeah `aspect-ratio` should paste over that. Unless you're on IE in which case you have bigger problems in your life